README.md

```diff
@@ -355,6 +355,21 @@ to be notified when the action completes:
 * `addNewFile([options, callback])` - adds a new text file, using the provided options, which can include: `filename` a `String` with the complete filename to use; `contents` a `String` with the new text file's data; `ext` a `String` with the new file's extension; `basenamePrefix` a `String` with the basename to use when generating a new filename.  NOTE: if you provide `filename`, `basenamePrefix` and `ext` are ignored.
 * `addNewFolder([callback])` - adds a new folder.
 * `export([callback])` - creates an archive `.zip` file of the entire project's filesystem, and downloads it to the browser.
+* `addCodeSnippet([options, callback])` - adds a new code snippet, using the provided option, which includes: `snippet` a `String` with the complete code snippet.
```

Comment:
Let's change this to:
`addCodeSnippet(snippet, [callback])` - adds a new code snippet to the editor (if it is in focus) at the current cursor position. One required parameter (`snippet`) needs to be passed in which needs to be a `String`.

And remove the example snippet (don't think it's necessary).